src/metabase/pulse/models/pulse_channel.clj

```diff
@@ -166,11 +167,6 @@
   (update-send-pulse-trigger-if-needed! pulse-id pulse-channel :remove-pc-ids #{(:id pulse-channel)}))
 
 ;; we want to load this at the top level so the Setting the namespace defines gets loaded
```

Comment:
We're no loading the namespace here. should we bring back some code to do it? or can we delete the comment?

Reply:
we can delete the command now the function is an defenterprise, we don't need to do that trick anymore.